crates/bitwarden/src/vault/cipher/login.rs

```diff
@@ -97,7 +97,26 @@ pub struct Fido2Credential {
     pub creation_date: DateTime<Utc>,
 }
 
-#[derive(Serialize, Deserialize, Debug, JsonSchema)]
+#[derive(Serialize, Deserialize, Debug, JsonSchema, Clone)]
+#[serde(rename_all = "camelCase", deny_unknown_fields)]
+#[cfg_attr(feature = "mobile", derive(uniffi::Record))]
+pub struct Fido2CredentialView {
```

Comment:
Future todo. Should we expose all fido stuff decrypted?

Reply:
That's something I'm not sure yet, we might want to expose only some parts by having some kind of `Fido2CredentialListView`, like we do with others. This will probably change once we get to actually implement this.